composer.json

```diff
@@ -12,10 +12,10 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "fsi/admin-bundle" : "~1.1@dev",
-        "symfony/framework-bundle": "~2.3",
```

Comment:
because of that there is no sense in keeping lowest dependencies support in admin-bundle.